gradle/geode-dependency-management.gradle

```diff
@@ -18,146 +18,245 @@
+        dependency('org.assertj:assertj-core:3.11.1')
+        dependency('org.awaitility:awaitility:3.1.2')
+        dependency('org.bouncycastle:bcpkix-jdk15on:1.60')
+        dependency('org.codehaus.cargo:cargo-core-uberjar:1.6.10')
+        dependency('org.eclipse.persistence:javax.persistence:2.2.1')
+        dependency('org.fusesource.jansi:jansi:1.17.1')
+        dependencySet(group: 'org.hamcrest', version: '1.3') {
+          entry 'hamcrest-all'
+          entry 'hamcrest-core'
+          entry 'hamcrest-library'
+        }
+        dependency('org.httpunit:httpunit:1.7.3')
+        dependency('org.iq80.snappy:snappy:0.4')
+        dependency("org.jgroups:jgroups:${project.'jgroups.version'}")
+        dependency('org.mockito:mockito-core:2.23.0')
+        dependency('org.mortbay.jetty:servlet-api:3.0.20100224')
+        dependency('org.postgresql:postgresql:42.2.2')
+        dependencySet(group: 'org.powermock', version: '2.0.0-beta.5') {
+          entry 'powermock-api-mockito2'
+          entry 'powermock-core'
+          entry 'powermock-module-junit4'
+        }
+        dependencySet(group: 'org.seleniumhq.selenium', version: '3.13.0') {
+          entry 'selenium-api'
+          entry 'selenium-chrome-driver'
+          entry 'selenium-remote-driver'
+          entry 'selenium-support'
+        }
+        dependency('org.skyscreamer:jsonassert:1.5.0')
+        dependency("org.slf4j:slf4j-api:${project.'slf4j-api.version'}")
+        dependency('org.springframework.hateoas:spring-hateoas:0.25.0.RELEASE')
+        dependency('org.springframework.ldap:spring-ldap-core:2.3.2.RELEASE')
+        dependencySet(group: 'org.springframework.security', version: '4.2.8.RELEASE') {
+          entry 'spring-security-config'
+          entry 'spring-security-core'
+          entry 'spring-security-ldap'
+          entry 'spring-security-test'
+          entry 'spring-security-web'
+        }
+        dependency('org.springframework.shell:spring-shell:1.2.0.RELEASE')
+        dependencySet(group: 'org.springframework', version: '4.3.20.RELEASE') {
+          entry 'spring-aspects'
+          entry 'spring-beans'
+          entry 'spring-context'
+          entry 'spring-core'
+          entry 'spring-expression'
+          entry 'spring-oxm'
+          entry 'spring-test'
+          entry 'spring-tx'
+          entry 'spring-web'
+          entry 'spring-webmvc'
+        }
+        dependency('pl.pragmatists:JUnitParams:1.1.0')
+        dependency('redis.clients:jedis:2.9.0')
+        dependency('xerces:xercesImpl:2.12.0')
+      }
+    }
+  }
+
+  void set_extended_property_versioning(Project project) {
+    // These versions are consumed beyond the scope of source set dependencies.
+    project.ext {
+      // These version numbers are consumed by :geode-modules-assembly:distAppServer filtering
+      // Some of these are referenced above as well
+      set('antlr.version', '2.7.7')
```

Comment:
I like this much better than the plain-text file holding the name-value pairs of package to version.